.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I've got a six-month old eslint config that I've liked, though will need to edit to do ts parsing. IIRC we will still need `typescript-eslint-parser` and `eslint-plugin-typescript`

```javascript
module.exports = {
  extends: ['airbnb', 'prettier', 'prettier/react'],
  plugins: ['prettier'],
  parser: 'babel-eslint',
  env: {
    browser: true,
  },
  rules: {
    // general js
    'import/default': 1,
    'import/named': 1,
    'import/namespace': [1, {allowComputed: true}],
    'import/prefer-default-export': 0,
    'import/no-extraneous-dependencies': [1, {devDependencies: true}],
    'import/order': [1, {'newlines-between': 'always'}],
    'no-confusing-arrow': 0,
    'no-underscore-dangle': 0,

    // prettier
    'prettier/prettier': [
      2,
      {
        printWidth: 100,
        tabWidth: 4,
        semi: false,
        singleQuote: true,
        trailingComma: 'es5',
        bracketSpacing: false,
      },
    ],
    
    // react-specific
    'react/require-default-props': 0,
    'react/no-unused-prop-types': 0,
    'react/jsx-filename-extension': 0,

    // custom
    'jsx-a11y/anchor-is-valid': [ 'error', {
      'components': ['Link'],
      'specialLink': ['to'],
      'aspects': ['noHref', 'invalidHref', 'preferButton']
    }],
  },
  settings: {
    'import/ignore': ['node_modules', '.(png|svg|jpg|css|pdf)$'],
    'import/extensions': ['.js'],
  },
}```

Reply:
Sure. I don't have strong preferences as long as it is consistent. No semi colons is going to take a bit of getting used to, but I'm up for trying it.

Reply:
Lint settings updated!